static/app/components/sidebar/isItemActive.tsx

```diff
@@ -0,0 +1,30 @@
+import {normalizeUrl} from 'sentry/utils/withDomainRequired';
+
+interface Item {
+  label?: React.ReactNode;
+  to?: string;
+}
+
+export function isItemActive(item: Item, exact?: boolean): boolean {
```

Comment:
Nit, but you could simplify the fn signature to something like `matchesSentryRoute(url: string, pathname: string) ` and call it like `matchesSentryRoute(item.to, location.pathname)`. That way it's decoupled of the entire sidebar item type and could easily get reused in different places if needed.

Reply:
I wanted to, but it has a bunch of custom logic related to the item's _label_ so it's actually very tightly coupled to the `Item` type after all!

Reply:
I might be misreading, but I think it just looks up the `item.to.split("?")[0]`, which would become `to.split("?")[0]` or even better - `new URL(to).pathname`. You would just drop the entire Item interface in that case

Reply:
I mean the `item?.label` checks that live a little lower in the function, I'd have to pass at a minimum the URL, the label, and `exact`. Which is fine if you want that, I just opted to keep using the `Item` type

Reply:
Ah right, no you are right. Idk how I missed that. Yeah, using Item type is fine then